protoc-gen-bq-json-schema/genjson/config.go

```diff
@@ -0,0 +1,11 @@
+package genjson
+
+import "github.com/spf13/pflag"
+
+type Config struct {
+	Indent bool
```

Comment:
same comment as above - let's make `Indent` a `string`!

Reply:
I had some issues with providing a string with just spaces as an `opt` in `buf.gen.yaml`. So I changed this to be called `Format` instead to make a distinction from the `Indent` options in `jsonbq.MarshalOptions` which is now a string.